code/modules/uplink/uplink_items.dm

```diff
@@ -1959,6 +1966,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	name = "Clown Costume"
 	desc = "Nothing is more terrifying than clowns with fully automatic weaponry."
 	item = /obj/item/storage/backpack/duffelbag/clown/syndie
+	
+/datum/uplink_item/badass/costumes/tactical_naptime
```

Comment:
u forgot to include only clown and nuke ops here

Reply:
It inherits it from 1953: all badass costumes are nuke and clown ops only.